navigate_through_poses_client_node:
  ros__parameters:
    poses: [pose0,pose1,pose2,pose3,pose4]
    #x, y, yaw(radians)
    pose0: [5.0,  -10.0,  2.09]
    pose1: [10.0, -11.0,  1.85]
    pose2: [15.0, -11.0,  1.44]
    pose3: [18.0, -11.0,  1.54]
    pose4: [20.0, -11.0, -0.53]